process/wg-lifecycle-documents/ai_ml_incubating_stage.md

```diff
@@ -0,0 +1,60 @@
+## AI/ML Working Group incubation application
+
+### List WG Chairs
+
+The WG must have a minimum of 1 Chair
+
+* Mihai Maruseac, Google, mihaimaruseac
+* Jautau "Jay" White, Microsoft, camaleon2016
+
+### Working Group (WG) has met all Sandbox requirement
+
+* Applying directly to Incubating
+
+### List of regular contributors
+
+The WG must have a minimum of 5 contributors from at least 3 different
+organizations attending regularly.
+
+* David A. Wheeler, LF
+* David Edelsohn, IBM / AI Alliance
+* Emily Fox, Red Hat
+* Jautau "Jay" White, Microsoft
+* Karen Bennet, ISO
+* Marcela Melara, Intel
+* Mihai Maruseac, Google
+* Newt Tan, University of Glasgow
+* Sarah Evans, Dell Technologies
+* Yotam Perkal, Rezilion
+
+and many others
+
+### Mission of the Working Group
+
+The WG must have a charter or mission statement for review by TAC
+
+* https://github.com/ossf/ai-ml-security/blob/main/mvsr.md
+
+### Governance
+
+WG must have documented, initial group governance.
+
+* https://github.com/ossf/ai-ml-security/blob/main/doc/CHARTER.md
```

Comment:
If you update the MVSR link above, the text on this charter should be updated as well.

Reply:
Updating it in https://github.com/ossf/ai-ml-security/pull/24